app/styles/partials/footer.scss

```diff
@@ -1,10 +1,15 @@
 .footer-pusher {
   display: flex;
   flex-direction: column;
-  overflow-x: hidden;
   min-height: 100vh;
 }
 
+#main-container {  // sass-lint:disable-line no-ids
```

Comment:
Why this?

Reply:
It is necessary due to the precedence of styling. There are other classes which are overriding the properties:

_**@media only screen and (min-width: 768px) and (max-width: 991px)
.ui.container {
    width: 723px;
    margin-left: auto!important;
    margin-right: auto!important;_**

Hence to override this id selector is the way. There is already a class named main-container but its been overridden by the media query and class .ui.container.

Reply:
@media only screen and (max-width: 767px)
.ui.container {
    margin-left: 1em!important;
    margin-right: 1em!important;
    
    Also, this media query is overriding the .main-container class for small screen devices.

Reply:
But why is this style necessary. Won't it break other pages?

Reply:
The parent container footer-pusher was overflowed by the main-container. I think Adding/overriding default margins (added by UI.container) will only prevent overflow but the parent container's dimensions will not be affected if used in other pages.

Reply:
Other pages (if they have any of these containers) will still have the default .ui.containers properties they are not overridden.